src/Angular.js

```diff
@@ -152,12 +154,19 @@ var minErrConfig = {
  * * `objectMaxDepth`  **{Number}** - The max depth for stringifying objects. Setting to a
  *   non-positive or non-numeric value, removes the max depth limit.
  *   Default: 5
+ *
+ * * `urlMaxLength`  **{Number}** - The max length of the URL reference in the error messages, it can be any number more than or equal 200. Setting to a
```

Comment:
If there is no good reason for the 200 limit. Let's remove it.
And then we can reuse `isValidObjectMaxDepth()` for both validations (under a different, more generic name :wink:).